frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js

```diff
@@ -111,6 +107,26 @@ describe("LeaderboardTable tests", () => {
     expect(screen.getAllByText("$1,000.00")[0]).toHaveStyle("text-align: right;");
 
   });
+
+  test("Cows owned is formatted correctly", () => {
+    const currentUser = currentUserFixtures.adminUser;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <LeaderboardTable leaderboardUsers={leaderboardFixtures.fiveUserCommonsLB} currentUser={currentUser} />
```

Comment:
If you remove a prop from a component, also remove any existing components that pass the prop in. There's a couple more.
```suggestion
    render(
      <QueryClientProvider client={queryClient}>
        <MemoryRouter>
          <LeaderboardTable leaderboardUsers={leaderboardFixtures.fiveUserCommonsLB} />
```

Reply:
@chriscerie raises a good point.  Since `currentUser` is no longer a prop of `LeaderboardTable`, we should no longer pass it in.  Please check for this.